tests/test_runner/test_discover_runner.py

```diff
@@ -396,3 +401,9 @@ def test_serialize(self):
             'test_runner_apps.databases.tests.DefaultDatabaseSerializedTests'
         ])
         self.assertEqual(databases, {'default': True})
+
+    def test_nested_suite(self):
+        databases, _ = self.get_databases([
+            'test_runner.test_discover_runner.get_suite',
```

Comment:
Thanks for fixing this.

I would add a comment here that you're relying on [`loadTestsFromName()`'s ability](https://docs.python.org/3/library/unittest.html#unittest.TestLoader.loadTestsFromName) to interpret a "name" (aka test label) as resolving to "a callable object that returns a `TestCase` or `TestSuite` instance." Otherwise, it's a bit obscure. For example, this case isn't acknowledged in Django's `build_suite()` documentation:
https://docs.djangoproject.com/en/3.1/topics/testing/advanced/#django.test.runner.DiscoverRunner.build_suite
nor inside `build_suite()`'s comments itself:
https://github.com/django/django/blob/807226cd79dbe61a8b3ba9fdfae865c5685f0b42/django/test/runner.py#L573

I didn't understand why this test was working at first because the string isn't a `TestCase` class, method, or module, etc. like in the other tests in this class.